app/views/calculators/adjustable_income/_estimate.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id="js-estimate">
+  <p>
+    Based on what you’ve told us, if you use your <%= number_to_currency(pot) %> pension pot to take an adjustable income, you could get:
+
+    <strong class="calculator__estimate__number t-calculator-tax-free-lump-sum">
+      <%= number_to_currency(estimate[:tax_free_lump_sum]) %> tax free
+    </strong>
+
+    and from the <span class="t-calculator-taxable-portion"><%= number_to_currency(estimate[:taxable_portion]) %></span> left over:
+
+    <strong class="calculator__estimate__number">
+      a monthly income of <span class="calculator__highlight"><%= number_to_currency(monthly_drawdown_amount) %></span>
+      would last<br>until you're <span class="t-calculator-age-for-desired-income calculator__highlight"><%= estimate[:desired_income_with_pot_growth_lasts_until] %></span>
+    </strong>
+  </p>
+</div>
+
+<p>
+  <label for="slider">or try with a different monthly income:</label>
+  <%= render 'components/slider', {
+      id: 'slider',
+      target: '#desired_monthly_income',
+      option: 'adjustable-income',
+      text_before: '£',
+      initial_value: monthly_drawdown_amount,
+      min: 50,
+      max: monthly_drawdown_amount.to_f * 5,
```

Comment:
Should we add logic so that the min value can not result in payments until your 150. :)